adafruit_shell.py

```diff
@@ -369,6 +373,31 @@ def copy(self, source, destination):
                     destination += os.sep + os.path.basename(source)
                 shutil.copy(source, destination)
 
+    def chmod(self, location, mode):
```

Comment:
May want to verify `mode` type, https://docs.python.org/3.11/library/os.html#os.chmod

Reply:
Yeah, I wanted either an octal value in the range of 0o0 to 0o777 or the flags (which should result in an octal value in the same range), so at least a value range is a good check to add.